src/components/AdoptionProcess/ProcessContainer.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import ProcessItems from "./ProcessItems";
+import { Container, Row, Col } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
this shouldnt be improted here it should be in the app.js

Reply:
done